_posts/it/2020-10-16-ALTF4-9.md

```diff
@@ -0,0 +1,84 @@
+
+### Underneathies
+
+Questo termine che abbiamo tutti sentito usare da player rinomati (come il nostro amato [KatherineOfSky](https://www.youtube.com/channel/UCTIV3KbAvaGEyNjoMoNaGtQ)) riferendosi alle [underground belts](https://wiki.factorio.com/Underground_belt), viene utilizzato anche su Reddit. Un'esempio è quando [/u/excessionoz](https://www.reddit.com/user/excessionoz/) ha accidentalmente [distrutto il suo mall](https://www.reddit.com/r/factorio/comments/9s7x30/tip_be_very_sure_of_your_circumstances_when/) (un altro esempio di gergo) utilizzando mod [Upgrade Planner](https://mods.factorio.com/mod/upgrade-planner). Ma da dove viene questa strana parola?
+
+Tutto iniziò il 9 marzo 2017. Questo fatidico giorno, [/u/Dynamic_Gravity](https://www.reddit.com/user/Dynamic_Gravity/) ha vuto una ~~discussione~~ con la sua ragazza riguardo come veramente fossero chiamate le underground belts. Non riuscendo a trovare un accordo, hanno deciso di chiedere agli *esperti* su Reddit creando un [sondaggio](https://www.reddit.com/r/factorio/comments/5yi071/need_your_help_in_settling_a_debate_with_gf/). E *quel* sondaggio, così avidamente difeso dalla sua ragazza, aveva tra le opzioni la parola _underneathies_.
+
+E, come si dice, il resto è storia. La community ha prontamente implementato questa parola nel gergo comune. Solo tre giorni dopo c'era già una [mod](https://mods.factorio.com/mod/Underneathies) per cambiare il nome dell'oggetto in-game (creata da [/u/Suprcheese](https://www.reddit.com/user/Suprcheese/), lo stesso che ci ha regalato mod leggendarie come la [Squeak Through](https://mods.factorio.com/mod/Squeak%20Through) e la [Orbital Ion Cannon](https://mods.factorio.com/mod/Orbital%20Ion%20Cannon)). Tre anni dopo questa parola è ancora utilizzata e amata da molti.
+
+E' affascinante osservare con che naturalezza il vocabolo sia entrato nel parlare della community. Capitemi, potremmo benissimo averle chiamate [_undie-grundies_](https://www.reddit.com/r/factorio/comments/5yi071/need_your_help_in_settling_a_debate_with_gf/deqgd0x?context=3). Abbiamo decisamente fatto la scelta giusta.
+
+### Nauvis: La Nosra Casa
+
+Vaste pianure verdi, pericolosi deserti rossi e laghi senza fondo. Una terra ricca di risorse ma abitata dalla minaccia biter. Questo è il mondo che abbiamo imparato ad amare dopo essercisi  brutalmente schianti sopra. E' la nostra casa, e la chiamiamo Nauvis.
+
+{% include figure.html src='https://media.alt-f4.blog/ALTF4/9/nauvis_day.jpg' alt='Foto di Nauvis di Giorno' caption='I crediti per l'immagine sono della mod <a href="https://mods.factorio.com/mod/NauvisDay">Nauvis Day</a>, con dell'editing fatto dall'autore.' %}

```

Comment:
`caption='I crediti per l'immagine sono della mod <a href="https://mods.factorio.com/mod/NauvisDay">Nauvis Day</a>, con dell'editing fatto dall'autore.'`
it will break due to apostrophe `'` symbols inside of string. Because of this we have to use special escape code `&#39;` instead of apostrophe symbol.

```
{% include figure.html src='https://media.alt-f4.blog/ALTF4/9/nauvis_day.jpg' alt='Foto di Nauvis di Giorno' caption='I crediti per l&#39;immagine sono della mod <a href="https://mods.factorio.com/mod/NauvisDay">Nauvis Day</a>, con dell'editing fatto dall&#39;autore.' %}
```

Reply:
ohhh

Reply:
you just updated it 
XD
well newer mind then.

Reply:
^^